Traceback (most recent call last):
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\users\natsciteaching\anaconda3\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\nbclient\client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\users\natsciteaching\anaconda3\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
## Setup

# Import all packages for the workshop
import numpy as np
import matplotlib.pyplot as plt
import matplotlib.image as mpimg
import scipy.ndimage as sn

# Load example image
A = mpimg.imread("bw.png")
A = np.mean(A, axis=2)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mImportError[0m                               Traceback (most recent call last)
[1;32mC:\Users\NATSCI~1\AppData\Local\Temp/ipykernel_26660/2540240674.py[0m in [0;36m<module>[1;34m[0m
[0;32m      5[0m [1;32mimport[0m [0mmatplotlib[0m[1;33m.[0m[0mpyplot[0m [1;32mas[0m [0mplt[0m[1;33m[0m[1;33m[0m[0m
[0;32m      6[0m [1;32mimport[0m [0mmatplotlib[0m[1;33m.[0m[0mimage[0m [1;32mas[0m [0mmpimg[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 7[1;33m [1;32mimport[0m [0mscipy[0m[1;33m.[0m[0mndimage[0m [1;32mas[0m [0msn[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      8[0m [1;33m[0m[0m
[0;32m      9[0m [1;31m# Load example image[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\anaconda3\lib\site-packages\scipy\ndimage\__init__.py[0m in [0;36m<module>[1;34m[0m
[0;32m    151[0m [1;32mfrom[0m [1;33m.[0m[0mfilters[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m
[0;32m    152[0m [1;32mfrom[0m [1;33m.[0m[0mfourier[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 153[1;33m [1;32mfrom[0m [1;33m.[0m[0minterpolation[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    154[0m [1;32mfrom[0m [1;33m.[0m[0mmeasurements[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m
[0;32m    155[0m [1;32mfrom[0m [1;33m.[0m[0mmorphology[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\anaconda3\lib\site-packages\scipy\ndimage\interpolation.py[0m in [0;36m<module>[1;34m[0m
[0;32m     35[0m [1;32mfrom[0m [0mnumpy[0m[1;33m.[0m[0mcore[0m[1;33m.[0m[0mmultiarray[0m [1;32mimport[0m [0mnormalize_axis_index[0m[1;33m[0m[1;33m[0m[0m
[0;32m     36[0m [1;33m[0m[0m
[1;32m---> 37[1;33m [1;32mfrom[0m [0mscipy[0m [1;32mimport[0m [0mspecial[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     38[0m [1;32mfrom[0m [1;33m.[0m [1;32mimport[0m [0m_ni_support[0m[1;33m[0m[1;33m[0m[0m
[0;32m     39[0m [1;32mfrom[0m [1;33m.[0m [1;32mimport[0m [0m_nd_image[0m[1;33m[0m[1;33m[0m[0m

[1;32m~\anaconda3\lib\site-packages\scipy\special\__init__.py[0m in [0;36m<module>[1;34m[0m
[0;32m    641[0m [1;32mfrom[0m [1;33m.[0m[0msf_error[0m [1;32mimport[0m [0mSpecialFunctionWarning[0m[1;33m,[0m [0mSpecialFunctionError[0m[1;33m[0m[1;33m[0m[0m
[0;32m    642[0m [1;33m[0m[0m
[1;32m--> 643[1;33m [1;32mfrom[0m [1;33m.[0m [1;32mimport[0m [0m_ufuncs[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    644[0m [1;32mfrom[0m [1;33m.[0m[0m_ufuncs[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m
[0;32m    645[0m [1;33m[0m[0m

[1;31mImportError[0m: DLL load failed: The specified module could not be found.
ImportError: DLL load failed: The specified module could not be found.

